### ISSUE TITLE
Error in package dependencies during bootstrap.sh

### ISSUE THREAD

**Issue by fogs** [solidnerd/terraform-k8s-hcloud#17]
```
apt-get -q install -y kubelet kubeadm
Reading package lists...
Building dependency tree...
Reading state information...
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 kubeadm : Depends: kubernetes-cni (= 0.6.0) but 0.7.5-00 is to be installed
 kubelet : Depends: kubernetes-cni (= 0.6.0) but 0.7.5-00 is to be installed
E: Unable to correct problems, you have held broken packages.
```

**Comment by solidnerd** [comment]
Hey @fogs,

which are the current values of `master_image` and `node_image` on your setup ? It seems that you use an older ubuntu version. I think it can be fixed with `ubuntu-18.04` as value. 

We should use this `ubuntu-18.04` default now.

**Comment by fogs** [comment]
I cloned `master` from your git repo and did not change anything - looks like this defaults to `ubuntu-16.04`. This error went away when I added `-var kubernetes_version=1.16.2` to the command line.

**Comment by solidnerd** [comment]
This will be included it in 0.3.1 so we can lose this issue for now
